src/components/NewTask.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import "./NewTask.module.css";
+import { useState } from "react";
+
+const NewTask = () => {
+  const [tasks, setTasks] = useState([]);
+  const [name, setName] = useState({});
+
+  function registerTask(e) {
+    e.preventDefault();
+    const task = { name: name, status: "not done" };
+    console.log(e.target);
```

Comment:
Don't forget to remove it later

Reply:
Thanks for the information, I'll take it off!